src/__experimental__/components/select/select-list.component.js

```diff
@@ -111,4 +141,17 @@ class SelectList extends React.Component {
   }
 }
 
+SelectList.propTypes = {
```

Comment:
Would you like to add descriptions to the props like we have in the others components?

Reply:
Added in 1418cb4 :thumbsup: